src/components/MobileProject/MoblileProject.module.scss

```diff
@@ -0,0 +1,64 @@
+@import "../../assets/styles/theme";
+.projectImg {
+    width: 100%;
+    height: 273px;
+}
+.moblieProjectTitle{
+    color: $white;
+    font: 100% $helvetica;
+    font-size: 32px;
+    text-align: center;
+    margin-bottom: 32px;
+    margin-top: 80px;
+}
+.moblieProject{
+    font: 100% $helvetica; 
+    text-align: center;
+    background-color: $black_100_35;
+    color: $white;
+    margin: 8px;
```

Comment:
jak dajesz margin na 8px a potem bottom i top na inny to ściągnij to do po prostu `margin: 0px 8px;`